src/Arcus.Observability.Telemetry.Core/ContextProperties.cs

```diff
@@ -64,6 +64,15 @@ public static class RequestTracking
             public const string ResponseStatusCode = "ResponseStatusCode";
             public const string RequestDuration = "RequestDuration";
             public const string RequestTime = "RequestTime";
+            
+            public static class ServiceBus
+            {
+                public const string Endpoint = "ServiceBus-Endpoint";
+                public const string EntityName = "ServiceBus-Entity";
+                public const string EntityType = "ServiceBus-EntityType";
+
+                public const string DefaultOperationName = "Process";
+            }
```

Comment:
Nested public classes, that's a bit of a smell, don't you think ?
Do these classes need to be publicanyway ?  To me, it seems that they're only used internally ?  (Maybe I'm wrong)

Reply:
The purpose of those nested classes is to have a kindoff namespace and get more organization on our constants. I don't consider it a smell in this case. 😄 There're used in our Application Insights package to get them back out, I guess. And also, to make it possible for consumers to write their own enrichers and have easy access to the telemetry keys used in the telemetry context.

Reply:
Ok.